### ISSUE TITLE
update rule: react-a11y-img-has-alt - should allow role='presentation' with non-empty alt text.

### ISSUE THREAD

**Issue by ipip2005** [Microsoft/tslint-microsoft-contrib#298]
There is a specific scenario that I pasted in the issue  https://github.com/Microsoft/tslint-microsoft-contrib/issues/298
we should allow role='presentation' appears with non-empty alt text. Like

```
<img
  aria-hidden={ true }
  role='presentation'
  src='...'
  alt='<non-empty> alt text'
/>
```
In following situation:
```

<img alt='123'/>
<span>123</span>.
```
Sometimes the alt text equals to the nearest text. The alt text is no need to be discoverable by assistive technologies. But they should exist to show for sighted user once this img fails loading.
For screen reader users, they only be able to 'see' one text around (e.g. there is a img with alt='apple' and there is a span text near it <span>apple</apple>, than screen reader users only see one of them)
For sighted users, they both see the img and the span, but in the situation that the img fails loading, sighted users should know `which image failed loading` by seeing the alt text at the position of that image.
This is for a better support for both screen reader users and sighted users.
So the following format should be acceptable:

```
<img aria-hidden='true' role='presentation' alt='123' />
<span>123</span>
```
